beaconclient/prod_beacon_instance.go

```diff
@@ -179,7 +182,7 @@ func (c *ProdBeaconInstance) SyncStatus() (*SyncStatusPayloadData, error) {
 	uri := c.beaconURI + "/eth/v1/node/syncing"
 	timeout := 5 * time.Second
 	resp := new(SyncStatusPayload)
-	_, err := fetchBeacon(http.MethodGet, uri, nil, resp, &timeout, http.Header{}, false)
+	_, err := fetchBeacon(http.MethodGet, uri, nil, resp, &http.Client{Timeout: timeout}, http.Header{}, false)
```

Comment:
**question:**

why do we create a new client (and as a result a new TCP connection per each request) only for `/eth/v1/node/syncing` path?

Reply:
It's just to configure the timeout I think, will update to have 1 client in use for everything with a 5 second timeout (is that enough for the validator request?)